scripts/globals/helm.lua

```diff
@@ -1535,6 +1535,20 @@ xi.helm.onTrade = function(player, npc, trade, helmType, csid, func)
                 movePoint(player, npc, zoneId, info)
             end
 
+            if
```

Comment:
Note to future self: Move some magian functions to utils to make this nested check unnecessary